### DISCUSSION TITLE
Drag-and-drop

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#2783] [General]
**Português:** No teste da nova versão, a opção de arrastar e soltar não está funcionando. Já testei em duas máquinas diferentes e apenas a opção de clicar para navegar funciona.

**Inglês:** In testing the new version, the drag-and-drop option is not working. I have tested it on two different machines, and only the click-to-browse option works.

![image](https://github.com/user-attachments/assets/c3f2513e-8c38-4070-959f-d6e796530f62)

**Comment by mtdvlpr** [comment]
Duplicate of #2758

**Reply by e-Feitosa** [reply]
Understood, is that I had posted much earlier in the other repertoire, and even commented on this post there informing that I had already posted.
